### DISCUSSION TITLE
SideDownload error:

### DISCUSSION THREAD

**Discussion by Aliceewg** [aclap-dev/video-downloadhelper#468] [Help and Q&A]
### Your question

I can't download some videos.

[in#1 @ 0000018f3f059900] Error opening input: Server returned 403 Forbidden (access denied)
Error opening input file https://vod-akm.play.hotmart.com/video/VzZJwvgyR6/hls/VzZJwvgyR6-1601384686000-audio_2=128368.m3u8?hdntl=exp=1715349223~acl=/*~data=hdntl~hmac=e8b93eebd6c3a7136f246494e6b40693df3b809ee699c21c195dbd23ef3ec72a&app=f57b1bb2-30e3-4fd0-9c59-cfe00909f280&tokenParams=eyJhbGciOiJub25lIn0.eyJ0cmFuc2NvZGVkT2JqZWN0S2V5cyI6WyJ2aWRlby9WelpKd3ZneVI2L21wNC83MzQyNzI1Yy1jN2ZhLTQ5MzYtOGViNS04NjZmMTlhY2I3MGIubXA0IiwidmlkZW8vVnpaSnd2Z3lSNi9tcDQvOGVkNjBmYzAtZDM5OS00Njg2LTlkMGItYWVjNzMzNmVlODJkLm1wNCIsInZpZGVvL1Z6Wkp3dmd5UjYvbXA0Lzk5ZjYyN2Q2LWM2ZWQtNDg0OC05NjVmLWI5MGQ1NmY3ZjEwZi5tcDQiLCJ2aWRlby9WelpKd3ZneVI2L21wNC8yODBiOGVhMS0yOWRjLTQyZGMtODRmOC0xOGYyMDEzOTRmOWYubXA0IiwidmlkZW8vVnpaSnd2Z3lSNi9tcDQvNzQ4ZjhkMzktZGRhYS00YTk2LWJjMjQtOWIyODA3MDI5NWJlLm1wNCJdLCJ1cGRhdGVEYXRlIjoxNjAxMzg0Njg2MDAwLCJjZG5VcmwiOiJodHRwczovL3ZvZC1ha20ucGxheS5ob3RtYXJ0LmNvbSIsImRybSI6ZmFsc2UsInN1YiI6IlZ6Wkp3dmd5UjYiLCJpYXQiOjE3MTI2NjUzNzZ9..
Error opening input files: Server returned 403 Forbidden (access denied)

### Addon & CoApp version and details

version: 8.2.2.9
channel: stable
build date: 2024-03-19
build options: linuxlic: false, noyt: true, browser: chrome
lang: pt-BR
license: accepted
key (only 16 first characters): addf8b8ff6524dee
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/124.0.0.0 Safari/537.36
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\Alice",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360955"
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

The second part of your problem report where you show your releases of software is just what is needed in a proper problem report.  Unfortunately, the first part of your problem report turns out to be useless.  When I tried to visit the URL in the error message, it told me I was not authorized to access that URL.  This is typical.  When you see a URL with long strings of gibberish like in that URL, it usually means that the URL is specific to your individual visit to that web site, & that it has a limited time until it times out.  This appears to be the case here.  I would not be surprised if the URL doesn't even work any more for you.  What we need instead is the URL of the web page on which the problematic video is hosted.  If you think multiple URLs would illustrate your problem better, go ahead & post them.  But we need the original URLs of web pages, nothing else.
